### ISSUE TITLE
Switch to Python 3

### ISSUE THREAD

**Issue by Draykan** [hjuutilainen/sus-inspector#23]
It would be great if SUS Inspector would start using Python 3 instead of the EOL Python 2 branch.
Thank you.

**Comment by W4JEW** [comment]
Agreed - Python 2 was killed off over a year ago at this point and, unless I missed the memo, it's not possible to run Python 2 on Apple Silicon. SUS Inspector is effectively useless to Apple Silicon users at this point.

**Comment by ehjmx** [comment]
You can get Python2 via MacPorts, Homebrew, macadmins or even python.org. Some of those are Apple Silicon native, some require Rosetta2. It's not drag and drop but not that hard to install them at one supported place or make the necessary edits to the plist files. It is documented [here](https://github.com/hjuutilainen/sus-inspector/releases) under "New features and changes" for version 2.1.
That being said I support a switch so Python 3 as well.

**Comment by LeeBinder** [comment]
Brew have removed python2 from their repository.

The install via pyenv of the latest v.2.x python 2.7.18 doesn't produce a working install anymore, either.

What has worked for me in Monterey is the procedure from https://stackoverflow.com/a/71513250

**Comment by W4JEW** [comment]
Clearly the developer has no interest in having people use the application. It's a shame - SUS Inspector could be very helpful! But it's worthless without supporting Python 3.

**Comment by LeeBinder** [comment]
I wouldn't call it "worthless" because it still works with P. v.2.7.18. But yes, antiquated and depreciated for sure.

**Comment by cipineda** [comment]
will this app get updated?
